server/.env.example

```diff
@@ -1,2 +1,3 @@
 DATABASE_URL="postgresql://user:password@localhost/database"
-DATABASE_SSL="false"
\ No newline at end of file
+DATABASE_SSL="false"
+API_KEY="1280U92JJNDLIAIJO"
```

Comment:
Is this your real `API_KEY`? Don't add it to repo as best practice

Reply:
this is the example file for the .env, so it's not the real thing.